frontend/src/ui/PathSegmentInput.tsx

```diff
@@ -21,18 +21,15 @@ export const PathSegmentInput = React.forwardRef<HTMLInputElement, Props>(
             gap: 8,
             backgroundColor: "var(--grey97)",
         }}>
-            <span css={{ paddingLeft: 8 }}>{base + (base.endsWith("/") ? "" : "/")}</span>
+            <span css={{ paddingLeft: 8, overflow: "auto" }}>
+                {base + (base.endsWith("/") ? "" : "/")}
+                <wbr />
```

Comment:
This just adds one `<wbr />` at the end which doesn't really do anything. I meant something like this:

```js
base.split("/").join("\u200b/") + (base.endsWith("/") ? "" : "/")
```

This adds the zero-width space unicode char before every `/` so that the browser can break there.